### PR TITLE
Explained where the CLI looks for global.json

### PR DESCRIPTION
**Pull request by svick**
The document explained what global.json is, but not where it can be located. This is especially important when it is in one of the parent directories or when calling `dotnet` from outside the project directory ([like this SO commenter was](http://stackoverflow.com/questions/40699867/choose-the-sdk-version-e-g-preview3-vs-preview2-that-the-dotnet-cli-uses/40708595?noredirect=1#comment73457646_40708595)).

I'm not happy about the "one of its parent directories" part (because a directory can have at most one parent), but I couldn't figure out anything else that is both succinct and accurate.

### CODE REVIEW

**Comment by Thealexbarney** [comment]
I would specify more explicitly that the global.json should be located in the current working directory and not the project directory. Calling a dotnet command on a project in a different directory won't use the global.json in the project's directory. I could see this tripping some people up.

Also, I've heard it referred to as the "Current working directory" or "Working directory", but not "Current directory". That might just be ignorance on my part, but more clarity probably couldn't hurt.

**Comment by svick** [comment]
@Thealexbarney Better?

File: docs/core/tools/global-json.md
Comment (GuardRex) [inline docs/core/tools/global-json.md]: *Nice add!* :rocket:

Small suggestions ...
* If you want to update it, the "global.json" should be in italics.
* If you want to update it, the JSON file should have two-space indents.
* Use contraction: "does not" :point_right: "doesn't"
* There's a comma splice there ... a few ways to fix it: Drop the parentheses from the phrase and place a comma before *which* (forming a dependent clause), or you could leave the parentheses and remove the comma before *or* (there are two objects of the preposition *in* there ... so no comma with the conjunction).

**Review by guardrex — CHANGES_REQUESTED** [review]

**Comment by Thealexbarney** [comment]
@svick Yeah, that's clearer. I think the language on "which does not have to be the project directory" sounds a bit weird to me. Maybe something like "which is not necessarily the same as the project directory" would be better. Thoughts?

**Comment by svick** [comment]
@GuardRex @Thealexbarney Updated.

File: docs/core/tools/global-json.md
Comment (GuardRex) [inline docs/core/tools/global-json.md]: Contraction: "is not" :point_right: "isn't"

**Comment by GuardRex** [comment]
Looks :eyes: great @svick ... I can merge with the tiny contraction update.

File: docs/core/tools/global-json.md
Comment (svick) [inline docs/core/tools/global-json.md]: @GuardRex Done.

**Review by guardrex — APPROVED** [review]

**Comment by Thealexbarney** [comment]
Looks good. Clears up the confusion I had a little while back about the placement of the global.json file.

**Comment by GuardRex** [comment]
Thanks ... excellent. Brief delay on merging ... minor CI adjustments today for the repo. @mairaw will merge asap when its safe to do so.

**Comment by jariq** [comment]
Wow! Addition of 1 sentence required 7 comments, 6 change requests and 4 commits. You guys should receive *nitpicking-grand-master* award :wink: Anyway I am the SO commenter mentioned in the first comment and I am really grateful for this change 👍

**Comment by GuardRex** [comment]
> nitpicking-grand-master award

Yes, I'm sorry about that. We have to adhere to the MS Style manual *on pain of death!* :skull:

:grinning:

**Comment by GuardRex** [comment]
Cool beans. We have the 🚦 green light on merging. Thanks!

**Pull request merged**
